core/dns.go

```diff
@@ -116,52 +80,67 @@ func (dnsResolver *DNSResolver) LookupTXT(hostname string) ([]string, time.Durat
 	return txt, rtt, err
 }
 
-// LookupHost uses a DNSSEC-enabled query to find all A/AAAA records associated with
-// the provided hostname. If the query fails due to DNSSEC, error will be
-// set to ErrorDNSSEC.
+// LookupHost sends a DNS query to find all A/AAAA records associated with
+// the provided hostname.
 func (dnsResolver *DNSResolver) LookupHost(hostname string) ([]net.IP, time.Duration, error) {
 	var addrs []net.IP
 	var answers []dns.RR
 
 	m := new(dns.Msg)
 	m.SetQuestion(dns.Fqdn(hostname), dns.TypeA)
-	r, aRtt, err := dnsResolver.LookupDNSSEC(m)
+
+	r, aRtt, err := dnsResolver.ExchangeOne(m)
 	if err != nil {
 		return addrs, aRtt, err
 	}
+	if r.Rcode != dns.RcodeSuccess && r.Rcode != dns.RcodeNameError && r.Rcode != dns.RcodeNXRrset {
+		err = fmt.Errorf("Failure at resolver: %d-%s for A query", r.Rcode, dns.RcodeToString[r.Rcode])
+		return nil, 0, err
+	}
```

Comment:
And for that matter, just have `ExchangeOne()` return an error if r.Rcode != dns.RcodeSuccess.